XamlControlsGallery/SettingsPage.xaml.cs

```diff
@@ -47,7 +47,13 @@ public SettingsPage()
             if (ElementSoundPlayer.SpatialAudioMode == ElementSpatialAudioMode.On)
                 spatialSoundBox.IsChecked = true;
             if (NavigationRootPage.Current.NavigationView.PaneDisplayMode == Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Top)

```

Comment:
In general, I'd expect the selected index to be set to 0 first instead of 1 as it is here.

Could this be reversed by doing:
```
if (NavigationRootPage.Current.NavigationView.PaneDisplayMode == Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Left)
{
    navigationLocation.SelectedIndex = 0;
}
else
{
    navigationLocation.SelectedIndex = 1;
}
```

Reply:
If this results in a cascade of changes, it's fine to have the order that is shown in the PR. From a purity point-of-view though, I think the order is upside down 🙃

Reply:
I've fixed the order now. We need to compare agains displaymode Auto here instead of left since we are actually using auto in the app. But I totally agree with you, this order looks better!